templates/Carbon/etc/fail2ban/jail.conf.epp

```diff
@@ -19,7 +19,7 @@
 
 [INCLUDES]
 
-before = paths-fedora.conf
+before = <%= scope['::fail2ban::before_file'] %>
```

Comment:
please mind that this is now an epp template and variables are accessed in a different way:
https://puppet.com/docs/puppet/5.5/lang_template_epp.html